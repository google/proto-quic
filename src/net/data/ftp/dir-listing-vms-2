
Directory SYS$SYSDEVICE:[ANONYMOUS]

ANNOUNCE.TXT;2           1/16         12-MAR-2005 08:44:57  [SYSTEM]               (RWED,RWED,RE,RE)
BOINC.DIR;1              1/16         29-DEC-2005 21:33:21  [SYSTEM]               (RWE,RWE,RE,RE)
BZIP2.DIR;1              1/16         27-SEP-2005 19:45:39  [SYSTEM]               (RWE,RWE,RE,RE)
CDRTOOLS.DIR;1           3/16         10-MAR-2005 17:31:44  [SYSTEM]               (RWE,RWE,RE,RE)
DIFFUTILS.DIR;1          1/16         23-JUN-2007 23:04:21  [SYSTEM]               (RWE,RWE,RE,RE)
DTSS_NTP.DIR;1           2/16         25-SEP-2000 21:03:28  [SYSTEM]               (RWE,RWE,RE,RE)
FIXREC.DIR;1             1/16         20-DEC-2003 10:57:22  [SYSTEM]               (RWE,RWE,RE,RE)
GNUPG.DIR;1              1/16          9-AUG-2006 02:11:51  [SYSTEM]               (RWE,RWE,RE,RE)
GZIP.DIR;1               1/16          5-JUL-2006 21:59:45  [SYSTEM]               (RWE,RWE,RE,RE)
INFO-ZIP.DIR;1          15/16         20-SEP-2004 21:27:27  [SYSTEM]               (RWE,RWE,RE,RE)
INPUT.DIR;1              1/16          4-MAR-1999 22:14:34  [UCX$NOBO,ANONYMOUS]   (RWE,RWE,RWE,RWE)
KERMIT.DIR;1             1/16         25-FEB-2006 12:22:34  [SYSTEM]               (RWE,RWE,RE,RE)
LOGIN.COM;2              1/16         28-SEP-2006 09:20:32  [SYSTEM]               (RWED,RWED,RE,RE)
MISC.DIR;1               6/16         12-DEC-1999 17:31:56  [SYSTEM]               (RWE,RWE,RE,RE)
MMK.DIR;1                1/16         30-SEP-2009 08:06:26  [SYSTEM]               (RWE,RWE,RE,RE)
MOZ_TEST.DIR;1           1/16          8-APR-2008 17:12:53  [SYSTEM]               (RWE,RWE,RE,RE)
MPACK.DIR;1              1/16         21-AUG-2009 10:28:57  [SYSTEM]               (RWE,RWE,RE,RE)
MTOOLS.DIR;1             1/16         14-MAR-2006 15:05:01  [SYSTEM]               (RWE,RWE,RE,RE)
OPENSSL.DIR;1            1/16         12-JAN-2009 08:42:56  [SYSTEM]               (RWE,RWE,RE,RE)
PGP.DIR;1                1/16         19-SEP-1999 16:39:04  [SYSTEM]               (RWE,RWE,RE,RE)
PICS.DIR;1	No privilege for attempted operation
QREADCD.DIR;1            1/16         29-SEP-2004 20:32:38  [SYSTEM]               (RWE,RWE,RE,RE)
RZSPINUP.DIR;1           1/16         24-JUL-2004 21:34:12  [SYSTEM]               (RWE,RWE,RE,RE)
TEST.DIR;1               1/16          5-NOV-2008 21:59:10  [SYSTEM]               (RWE,RWE,RE,RE)
VIM.DIR;1                1/16         30-APR-2005 16:32:56  [SYSTEM]               (RWE,RWE,RE,RE)
VMSTAR.DIR;1             1/16          7-JUN-2007 09:36:04  [SYSTEM]               (RWE,RWE,RE,RE)
WELCOME.TXT;3            1/16         12-MAR-2005 08:45:28  [SYSTEM]               (RWED,RWED,RE,RE)
WGET.DIR;1               3/16         17-AUG-1999 20:41:54  [SYSTEM]               (RWE,RWE,RE,RE)
WGET_TEST.DIR;1          1/16         13-JUN-2006 21:29:27  [SYSTEM]               (RWE,RWE,RE,RE)
WPUT.DIR;1               1/16          9-DEC-2004 20:16:46  [SYSTEM]               (RWE,RWE,RE,RE)

Total of 30 files, 53/464 blocks
