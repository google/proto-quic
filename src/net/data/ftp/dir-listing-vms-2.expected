-
announce.txt
512
2005
3
12
8
44

d
boinc
-1
2005
12
29
21
33

d
bzip2
-1
2005
9
27
19
45

d
cdrtools
-1
2005
3
10
17
31

d
diffutils
-1
2007
6
23
23
4

d
dtss_ntp
-1
2000
9
25
21
3

d
fixrec
-1
2003
12
20
10
57

d
gnupg
-1
2006
8
9
2
11

d
gzip
-1
2006
7
5
21
59

d
info-zip
-1
2004
9
20
21
27

d
input
-1
1999
3
4
22
14

d
kermit
-1
2006
2
25
12
22

-
login.com
512
2006
9
28
9
20

d
misc
-1
1999
12
12
17
31

d
mmk
-1
2009
9
30
8
6

d
moz_test
-1
2008
4
8
17
12

d
mpack
-1
2009
8
21
10
28

d
mtools
-1
2006
3
14
15
5

d
openssl
-1
2009
1
12
8
42

d
pgp
-1
1999
9
19
16
39

d
qreadcd
-1
2004
9
29
20
32

d
rzspinup
-1
2004
7
24
21
34

d
test
-1
2008
11
5
21
59

d
vim
-1
2005
4
30
16
32

d
vmstar
-1
2007
6
7
9
36

-
welcome.txt
512
2005
3
12
8
45

d
wget
-1
1999
8
17
20
41

d
wget_test
-1
2006
6
13
21
29

d
wput
-1
2004
12
9
20
16
